### PR TITLE
DOC: Fix format of the See Also descriptions

### PR DESCRIPTION
**Pull request by syutbai**
- [ ] does not close, but partially contributes to #23630
- [ ] tests added / passed
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

**Review by datapythonista — APPROVED** [review]
Thanks a lot for working on this @syutbai, looks great.

I'm not sure if git will be able to merge automatically the changes to the titles, as they are the same exact change. I'd wait until we have the conflict, in case it's not needed.

**Comment by datapythonista** [comment]
Looks like git manages the merge ok.

@syutbai can you fix the linting errors on travis, so we can merge:
```
./pandas/core/frame.py:354:5: W191 indentation contains tabs
./pandas/core/frame.py:354:5: E101 indentation contains mixed spaces and tabs
./pandas/core/frame.py:1923:80: E501 line too long (80 > 79 characters)
```

**Comment by syutbai** [comment]
@datapythonista Sorry for the tabbing issues.  I pushed the changes to my branch.  Do I need to do another pull request or are we good for merge now?  Thanks!

**Comment by datapythonista** [comment]
Updating the branch is good enough, no need to open a new PR.

I see travis is still red, can you take a look and see what is wrong? I'm in my phone and can't check myself right now.

**Comment by syutbai** [comment]
I see the following error but the script ends with 0:
``` 
3 Errors found:
	err desc
	err desc
	err desc
.docstring1: err desc
docstring1: err desc
docstring1: err desc
docstring2: err desc
docstring2: err desc
..EXECUTED
{"docstring1": {"errors": [["ER01", "err desc"], ["ER02", "err desc"], ["ER03", "err desc"]]}, "docstring2": {"errors": [["ER04", "err desc"], ["ER05", "err desc"]]}}.Series.foo: err desc
DataFrame.bar: err desc
Series.foobar: err desc
Series.foo: err desc

=============== 75 passed, 4 xfailed, 5 warnings in 3.48 seconds ===============
The command "ci/script_single.sh" exited with 0.
```

The other script ends with exit code 1 but I'm unable to tell where the error is:
```
=================== 5 passed, 12 deselected in 0.04 seconds ====================
Doctests interval classes DONE
Check that requirements-dev.txt has been generated from environment.yml
Check that requirements-dev.txt has been generated from environment.yml DONE
The command "ci/code_checks.sh" exited with 1
```

**Comment by TomAugspurger** [comment]
The line causing the failure is at https://travis-ci.org/pandas-dev/pandas/jobs/454288173#L2823

**Comment by TomAugspurger** [comment]
```
./pandas/core/frame.py:1923:74: W291 trailing whitespace
```

not sure about why the errors you posted aren't failing the build, but @datapythonista will look at that separately I think.

File: pandas/core/frame.py [hunk 1]
```diff
@@ -1918,9 +1918,10 @@ def to_stata(self, fname, convert_dates=None, write_index=True,
 
         See Also
         --------
-        pandas.read_stata : Import Stata data files
-        pandas.io.stata.StataWriter : low-level writer for Stata data files
-        pandas.io.stata.StataWriter117 : low-level writer for version 117 files
+        pandas.read_stata : Import Stata data files.
+        pandas.io.stata.StataWriter : Low-level writer for Stata data files.
+        pandas.io.stata.StataWriter117 : Low-level writer for version 117
```
Comment (datapythonista) [inline pandas/core/frame.py]:
```suggestion
        pandas.io.stata.StataWriter117 : Low-level writer for version 117
```

**Review by datapythonista — COMMENTED** [review]
Added a code suggestion removing the whitespace, so you should be able to fix it with a single click.

The error you shown is not really a test error. Is the output of a docstring validation that is being printed (need to check why, as it shouldn't). But that's not really a test failing, is just a "debug" information.

**Comment by syutbai** [comment]
I pushed the change.

How did you identify the file and line number with the issue?

**Comment by TomAugspurger** [comment]
Restarted the failing travis build. Not sure what's up with it.

**Comment by TomAugspurger** [comment]
All green. Thanks @syutbai!

**Pull request merged**